python/mxnet/image/detection.py

```diff
@@ -249,6 +249,7 @@ def _check_satisfy_constraints(self, label, xmin, ymin, xmax, ymax, width, heigh
         coverages = coverages[np.where(coverages > 0)[0]]
         if coverages.size > 0 and np.amin(coverages) > self.min_object_covered:
             return True
+        return False
```

Comment:
maybe `return coverages.size > 0 and np.amin(coverages) > self.min_object_covered`?